.github/workflows/release-hash.yml

```
@@ -2,25 +2,47 @@ name: Release Hash
 on:
   workflow_dispatch:
     inputs:
-      hash:
-        description: 'IPFS Hash (CID)'
-        required: true
-        default: ''
+      cid:
+        name: cid
+        description: IPFS Hash (if blank, pulled from build step)
+
+      signature:
+        name: Signature
+        description: Signature (if blank, pulled via Seacrest)
 jobs:
   build:
     name: Release Hash
     runs-on: ubuntu-latest
+    environment: Production
     steps:
-      - name: Checkout
+      - name: Checkout repository
         uses: actions/checkout@v2
 
       - name: Use Node.js ${{ matrix.node-version }}
         uses: actions/setup-node@v1
         with:
-          node-version: '16.x'
+          node-version: '15.x'
+
+      - name: Install Release Dependencies
+        run: yarn install --ignore-optional --ignore-platform
+
+      - name: Download artifact
+        uses: dawidd6/action-download-artifact@v2
+        with:
+          github_token: ${{secrets.GITHUB_TOKEN}}
+          workflow: build.yaml
+          workflow_conclusion: success
+          commit: ${{github.event.pull_request.head.sha}}
```

```suggestion
          branch: main
```
so that a random pull request doesn't affect the deploy when we run the action (we almost always want to deploy from latest main)